slang-gfx.h

```diff
@@ -221,6 +221,8 @@ class IShaderProgram: public ISlangUnknown
     x(R16G16_FLOAT, 4, 1) \
     x(R16_FLOAT, 2, 1) \
     \
+    x(R64_UINT, 8, 1) \
```

Comment:
This is going to break binary compatibility, that's causing the failures in Falcor.

Reply:
does our testing system not replace the header?

Reply:
so it seems we can't add R64.* support unless the compile process is changed for falcor (to not rely on pacman)?

Reply:
We use a prebuilt binary of Falcor to speed up CI process, and to make sure we don't break binary compatiblity.